src/PAModel/IR/IRStateHelpers.cs

```diff
@@ -18,6 +18,8 @@ internal static class IRStateHelpers
     {
         public const string ControlTemplateOverridableProperties = "OverridableProperties";
         public const string ControlTemplatePCFDynamicSchemaForIRRetrieval = "PCFDynamicSchemaForIRRetrieval";
+        public const string ControlTemplateHostTypePropertyName = "HostType";
```

Comment:
Is this still used anywhere? I only see it used in the unit test.

Reply:
You are right, used only in unit test.